src/Adapter/LegacyContext.php

```diff
@@ -81,7 +90,11 @@ public function getContext()
         if (null === static::$instance) {
             $legacyContext = Context::getContext();
 
-            if ($legacyContext && !empty($legacyContext->shop) && !isset($legacyContext->controller) && isset($legacyContext->employee)) {
+            // This getter initialises a legacy admin controller, in the process of migrating the layout the Context::controller shouldn't be created
+            // by this method, this responsibility lies in the ContextShopListener that create a bridge admin controller, so when Symfony layout is
+            // enabled we skip this step to avoid instantiating multiple legacy controllers
+            $symfonyLayoutEnabled = $this->symfonyLayoutFeature && $this->symfonyLayoutFeature->isEnabled();
```

Comment:
```suggestion
            $symfonyLayoutEnabled = $this->symfonyLayoutFeature?->isEnabled();
```

Not blocking :) 

https://php.watch/versions/8.0/null-safe-operator